File: DSA With Cpp/arrays/reverse_array.cpp
// Reverse the elements in the array
#include <iostream>
using namespace std;

int reverse_array(int arr[], int size)
{
    int i = 0;
    int j = size - 1;
    while (i < j)
    {
        int temp = arr[i];
        arr[i] = arr[j];
        arr[j] = temp;
        i++;
        j--;
    }
}

int main()
{
    int arr[] = {4, 2, 7, 8, 1, 2, 5};
    int size = sizeof(arr) / sizeof(arr[0]);
    reverse_array(arr, size);
    for (int i = 0; i < size; i++)
    {
        cout << arr[i] << " ";
    }
}

File: DSA With Python/arrays/reverse_array.py
# Reverse the elements in the array

def reverse_array(arr, size):
    i = 0
    j = size - 1
    while (i < j):
        temp = arr[i]
        arr[i] = arr[j]
        arr[j] = temp
        i = i + 1
        j = j - 1


arr = [4, 2, 7, 8, 1, 2, 5]
reverse_array(arr, len(arr))
print(arr)
